### DISCUSSION TITLE
nodejs update to latest lts

### DISCUSSION THREAD

**Discussion by anandmuthu1978** [renovatebot/renovate#9260] [Request Help] [answered]
Hello Experts,

I have just started using renovate and I am already hitting an obstacle. The runtime environment that I use is very sensitive to the nodejs package that is available. Currently the nodejs that is supported is version 10 & 12. When I tried the first renovate PR, it is suggesting a plug in to update to version 14. 
![image](https://user-images.githubusercontent.com/70018716/112206774-443cb180-8c51-11eb-961b-27bb0c5b5ad4.png)

How do I exclude the nodejs update or better yet specify which nodejs it should be upgraded to. I have repo's that uses nodejs 8/10 which I would like to update the dependency to 12.

Hope you can help. 
Thank you in advance.

Anand Muthu

**Comment by rarkins** [comment, marked as answer]
If you are on Node.js v12 then it's expected and correct that Renovate proposes an update to v14.

If any patch or minor releases for v12 become available then you'll still get a separate PR for those too.

If you are not ready to update to Node v14, your options are:
- Allow the PR to be created, then close it. This "ignores" the v14 update forever, or
- Add a packageRule for `node` with `allowedVersions` equal `<13`. This *hides* any major updates forever

I recommend ignoring instead of hiding, presuming that one day you will upgrade. Also recommend you set `dependencyDashboard=true` in your repo so that you get visibility about any updates you've hidden.

If you have repos on Node 8 or 10 then by default Renovate will recommend v14 and not recommend updates to v12. However if you configure `"separateMultipleMajor": true` for `node` then you'll get a separate PR for each major upgrade available, so you could choose to close/ignore the v14 one while accepting the v12 one.

Finally, please follow #9264 because it's relevant to anyone using older versions of Node.js.
